brands/craft/key_cutter.json

```diff
@@ -0,0 +1,12 @@
+{
+  "craft/key_cutter|Mister Minit": {
+    "tags": {
+      "brand": "Mister Minit",
+      "brand:wikidata": "Q1939269",
+      "brand:wikipedia": "de:Mister Minit",
+      "craft": "key_cutter",
+      "name": "Mister Minit",
+      "repair": "only"
```

Comment:
I feel a bit weird about the `repair=only` tag here.  Surely they aren't repairing keys that weren't cut properly?

Reply:
The problem is that reviewing the [documentation](https://wiki.openstreetmap.org/wiki/Tag:craft%3Dkey_cutter), it also offers sale of things. However, this brand does not sell, so I added ["repair = only"](https://wiki.openstreetmap.org/wiki/Key:repair) (instead of "yes").

Reply:
Edit: This only applies in shops, that kind of logic that generates some confusion, I will remove this tag.

Reply:
thanks, yeah looking here https://misterminit.co/  they seem like they are a global brand that does more than just cut keys.  I'd put them as a `shop=repair` and skip the `repair=*` tag.

Reply:
This label of "repair shop" is the best choice. I fixed with matchTags.